packages/settingeditor/src/settingseditor.tsx

```diff
@@ -0,0 +1,99 @@
+import { ISettingRegistry, Settings } from '@jupyterlab/settingregistry';
+import { IStateDB } from '@jupyterlab/statedb';
+import { ITranslator } from '@jupyterlab/translation';
+import { IFormComponentRegistry, ReactWidget } from '@jupyterlab/ui-components';
+import { PanelLayout, SplitPanel, Widget } from '@lumino/widgets';
+import React from 'react';
+import { PluginList } from './pluginlist';
+import { SettingsPanel } from './settingspanel';
+import { Message } from '@lumino/messaging';
+import { Dialog, showDialog } from '@jupyterlab/apputils';
+
+export class SimpleSettingsEditor extends Widget {
```

Comment:
We probably want to rename to something like `FormSettingsEditor` or `SettingsEditor`.